src/PublicApiAnalyzers/Core/Analyzers/xlf/PublicApiAnalyzerResources.es.xlf

```diff
@@ -189,12 +189,12 @@
       </trans-unit>
       <trans-unit id="RemovedApiIsNotActuallyRemovedMessage">
         <source>Symbol '{0}' is marked as removed but it isn't deleted in source code</source>
-        <target state="new">Symbol '{0}' is marked as removed but it isn't deleted in source code</target>
+        <target state="translated">El símbolo ' {0} ' está marcado como retirado, pero no se elimina en el código fuente</target>
```

Comment:
@cristianosuzuki77 can you verify the spacing around `'` characters is intentional?